system/apps_featured/117_gabuino/web/v1/examples.js

```diff
@@ -1142,3 +1142,33 @@ examples =
     '    return 0;\n' +
     '}\n'
 }
+
+document.getElementById('input-file')
+      .addEventListener('change', getFile)
+
+    function getFile(event) {
+	    const input = event.target
+      if ('files' in input && input.files.length > 0) {
```

Comment:
you can try `if (input?.files.length)`

Reply:
fixed and tested localy :cowboy_hat_face: